source/development/makefile.rst

```diff
@@ -89,21 +89,21 @@ Available flags
 +-----------------------+------------------+-----------------------------------------------------------------------------------------------------+
 | **V**                 | Display full compiler messages.                                                                                        |
 |                       +------------------+-----------------------------------------------------------------------------------------------------+
-|                       | Default value    | 0                                                                                                   |
+|                       | Default value    | no                                                                                                  |
 |                       +------------------+-----------------------------------------------------------------------------------------------------+
-|                       | Allowed values   | 0, 1                                                                                                |
+|                       | Allowed values   | yes, no                                                                                             |
```

Comment:
The allowed values to enable the flag are `YES`, `yes`, `y`, `Y` and `1`. Please specify all of them in this field. `no` is not an allowed value since it works like any other string passed as a value, it makes the flag to be ignored. Please, review the other options for doing the same changes.